topics/admin/slides/introduction.html

```diff
@@ -155,46 +155,43 @@
 
 Connecting Galaxy to clusters/HPC is covered in the advanced section.
 
-
 ---
 
-# How is Galaxy deployed?
-
-Current:
-
-- Ansible / `git clone https://github.com/galaxyproject/galaxy.git`
-- [Docker](https://github.com/bgruening/docker-galaxy-stable/)
-- *Framework dependencies* provided as Python "wheels", fetched at first startup with `pip`
-- *Tool dependencies* provided as Conda packages or legacy Tool Shed packages
```

Comment:
Maybe this can be rewritten as:

- Code
  - git
- Framework Dependencies
  - Python wheels, fetched at startup with `pip`
- Tool dependencies
  - Conda packages
  - Singularity & Docker containers